frontends/web/package.json

```diff
@@ -34,7 +34,7 @@
     "react-i18next": "14.1.2",
     "react-router-dom": "6.4.3",
     "react-select": "5.8.0",
-    "request-address": "0.0.6"
+    "request-address": "^0.0.9"
```

Comment:
nit: for production deps other deps don't use semver ranges, so we can explicitly decide and test new versions.

Reply:
Thanks! Updated